modules/cnf-performing-end-to-end-tests-running-the-tests.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-performing-platform-verification-latency-tests.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-performing-end-to-end-tests-running-the-tests_{context}"]
+= Running the latency tests
+
+Run the cluster latency tests to validate node tuning for your Cloud-native Network Functions (CNF) workload.
+
+[IMPORTANT]
+====
+**Always** run the latency tests with `DISCOVERY_MODE=true` set. If you don't, the test suite will make changes to the running cluster configuration.
+====
+
+[NOTE]
+====
+When executing `podman` commands as a non-root or non-privileged user, mounting paths can fail with `permission denied` errors. To make the `podman` command work, append `:Z` to the volumes creation; for example, `-v $(pwd)/:/kubeconfig:Z`. This allows `podman` to do the proper SELinux relabeling.
+====
+
+.Procedure
+
+. Open a shell prompt in the directory containing the `kubeconfig` file.
++
+You provide the test image with a `kubeconfig` file in current directory and its related `$KUBECONFIG` environment variable, mounted through a volume. This allows the running container to use the `kubeconfig` file from inside the container.
+
+. Run the latency tests by entering the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
+-e DISCOVERY_MODE=true registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
+/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
```

Comment:
-e LATENCY_TEST_RUN=true is missing here because otherwise the test will not run

Reply:
this seems missing also in the other versions

Reply:
so to make it a bit simpler, all the commands that expects to run the latency tests should have this parameter enabled.